### ISSUE TITLE
Any result compared to the real t5?

### ISSUE THREAD

**Issue by redpanda3** [The-OpenROAD-Project/asap5#2]
Is the pdk accurate enough? Since a7 v t7 has around 50% differences in synthesis by my experimental trial.

**Comment by maliberty** [comment]
I'm curious about your a7 comment - are you using the same synthesis tool in both cases?

**Comment by redpanda3** [comment]
Hi, Matt. The trial is occasional. One in Genus+asap7, and another in Synopsys+t7. So I have this question. How are the accuracy and completeness of this pdk now?

**Comment by maliberty** [comment]
What is the 50% difference in (area, timing, power)?  On how many test cases?  I haven't heard of such a large variation in general.

We don't use a5 in ORFS so I have no data on it.   My impression is it is less complete than a7.

**Comment by redpanda3** [comment]
The result is just about area by synthesis happened long ago. I will use t**7_baseat_svtt_0p75 and compare again, with both asap7 6 and asap7 7.5 track, test it again. Do you have any suggestions?

What is ORFS? OpenRoad Full Stack?

**Comment by maliberty** [comment]
ORFS = https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts

**Comment by redpanda3** [comment]
And is it possible to synthesis using genus or dc in asap5? I would have a comparison between a5 and t5.

**Comment by redpanda3** [comment]
> ORFS = https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts

Yeah, I'm also a big fan of openroad, openroad+skywater130 gives many people an opportunity to tapeout.

**Comment by maliberty** [comment]
> And is it possible to synthesis using genus or dc in asap5? I would have a comparison between a5 and t5.

I have no experience to say but I would hope so.

**Comment by redpanda3** [comment]
Thanks for sharing.
